crates/oxc_linter/src/rules/typescript/no_misused_new.rs

```diff
@@ -68,15 +70,17 @@ impl Rule for NoMisusedNew {
                  let TSTypeName::IdentifierName(id) = &type_ref.type_name &&
                  id.name == decl_name {
 
-                     ctx.diagnostic(NoMisusedNewInterfaceDiagnostic(sig.span));
+                     ctx.diagnostic(NoMisusedNewInterfaceDiagnostic(
+                      Span::new(sig.span.start, sig.span.start + 3)
```

Comment:
Should this be span of the `id`?

Reply:
By the way, where is the comparison to `new`?

Reply:
> Should this be span of the `id`?

The `id` here refers to the return type, not the method name.

> By the way, where is the comparison to new or constructor? TSConstructSignatureDeclaration?

The two are distinguished by the `TSConstructSignatureDeclaration` and the `TSMethodSignature`.